docs/Fault-Tolerant-Tachyon-Cluster.md

```diff
@@ -3,12 +3,28 @@ layout: global
 title: Fault Tolerant Tachyon Cluster
 ---
 
+# Fault Tolerance
+
+Fault Tolerance in Tachyon is based upon a multi-master approach where multiple master processes are
+run.  One of these processes is elected the leader and is used by all workers and clients as the primary 
+point of contact.  The other masters act as a standby using the shared journal to ensure that they maintain the same
```

Comment:
Also, it's better to explain how non-leaders keep the same metadata as current leader as we discussed in google group.